build.sbt

```diff
@@ -86,7 +86,7 @@ val commonSettings = Def.settings(
   startYear := Some(2014),
   licenses := Seq("Apache-2.0" -> url("http://opensource.org/licenses/Apache-2.0")),
   description := "Alpakka is a Reactive Enterprise Integration library for Java and Scala, based on Reactive Streams and Akka.",
-  crossScalaVersions := Seq(Scala212, Scala211, Scala213).filterNot(_ == Scala211 && Nightly),
+  crossScalaVersions := Seq(Scala212, Scala213),
```

Comment:
The `verifyDocs` task failed on the new `clustersharding` sub project because its dependencies don't exist for Scala 2.11.  Overriding `crossScalaVersions` in `clustersharding` to not include Scala 2.11 didn't let work, so instead I set Scala 2.12 and 2.13 as part of `commonSettings` and override `crossScalaVersions` to include 2.11 in subprojects where applicable for cross building.

Reply:
I also re-factored the statement to exclude 2.11 cross builds on Nightly builds, because I found it difficult to parse.

old

```
Seq(Scala212, Scala211, Scala213).filterNot(_ == Scala211 && Nightly)
```

new

```
crossScalaVersions := { if (Nightly) Seq(Scala212, Scala213) else Seq(Scala212, Scala211, Scala213) }
```